x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/optimizer/PhysicalPlanOptimizerTests.java

```diff
@@ -144,9 +148,8 @@
 import static org.elasticsearch.xpack.esql.core.type.DataType.GEO_POINT;
 import static org.elasticsearch.xpack.esql.parser.ExpressionBuilder.MAX_EXPRESSION_DEPTH;
 import static org.elasticsearch.xpack.esql.parser.LogicalPlanBuilder.MAX_QUERY_DEPTH;
-import static org.elasticsearch.xpack.esql.plan.physical.AggregateExec.Mode.FINAL;
-import static org.elasticsearch.xpack.esql.plan.physical.AggregateExec.Mode.PARTIAL;
```

Comment:
You might want to check your IDE settings - there's no reason to remove this static imports; it just creates a bunch of noise for no good reason.

Reply:
I'll reduce the noise via static imports. Do note that this change is not artificial, though; `AggregateExec` now uses the `AggregatorMode` enum instead of its own `AggregateExec.Mode`, so we can represent `Intermediate` aggregations as well, and the two enums don't diverge anymore.